test/jdk/valhalla/valuetypes/StaticFactoryTest.java

```diff
@@ -67,8 +68,10 @@ public static void testPrimitiveClassConstructor() throws Exception {
     @Test
     public static void constructor() throws Exception {
         Constructor<?> ctor = PRIMITIVE_TYPE.getDeclaredConstructor();
-        Object o = ctor.newInstance();
-        assertTrue(o.getClass() == PRIMITIVE_TYPE.asPrimaryType());
+        for (int i = 0; i < 20; ++i) {
+            Object o = ctor.newInstance();
+            assertTrue(o.getClass() == PRIMITIVE_TYPE.asPrimaryType());
+        }
```

Comment:
I suggest to replace this change with `@run testng/othervm -Dsun.reflect.noInflation=true StaticFactoryTest` so that other test cases will also verify when it's run with the generated bytecode.